docs/guides/permissions/overview.mdx

```diff
@@ -198,7 +203,7 @@ ory is allowed User:Henning edit File private
 Allowed
 ```
 
-## Further reading
+## Next steps
 
-To learn more about the Ory Permission Language, read the [specification](../../keto/reference/ory-permission-language.mdx)
-document.
+- Learn [what is a permission model and how to create one](../../keto/modeling/create-permission-model.mdx).
```

Comment:
```suggestion
- Learn [what a permission model is and how to create one](../../keto/modeling/create-permission-model.mdx).
```

Reply:
Should we motivate this some more to put the reader on the right track? Something like:

Now you know the basics of working with relationships and permissions in Ory Permissions. Next, make it yours by learning what a permission model is and how to create one.

WDYT?

Also what's really missing is how to integrate this into your app, but that's what @zepatrik is working on, right?